masonite/providers/CsrfProvider.py

```diff
@@ -0,0 +1,19 @@
+""" A Csrf Service Provider """
+from masonite.provider import ServiceProvider
+from masonite.auth.Csrf import Csrf
+
+
+class CsrfProvider(ServiceProvider):
+
+    wsgi = True
+
+    def register(self):
+        request = self.app.make('Request')
+        self.app.bind('Request', request)
```

Comment:
remove the `self.app.bind('Request', request)`

the request is already binded to the container in the `AppProvider`

You're overwriting it here